app/enterprise/0.36-x/deployment/installation/free-trial.md

```diff
@@ -0,0 +1,230 @@
+---
+title: How to Install a Kong Enterprise Free Trial with Docker
+---
+
+## Introduction
+
+This guide walks through downloading, installing, and starting a Kong Enterprise
+Free Trial using Docker. Your Free Trial license is active for 15 days. If you
+would like to extend your trial please contact your **Sales** associate.
+
+
+## Prerequisites
+
+To complete this guide you will need:
+
+- A valid [Kong Enterprise Free Trial](https://konghq.com/free-trial)
+- Docker
+- A Kong Enterprise Free Trial docker file (`.tar.gz`)
+- A valid Kong Enterprise Free Trial license file
+
+
+## Step 1. Load the Kong Enterprise Free Trial Docker Image
+
+1. Download the Kong Enterprise Trial Image file
+
+    The `kong-enterprise-edition-ft-0.36.tar.gz` file can be downloaded by
+    clicking the Docker link in the Kong Enterprise Free Trial Welcome email.
+    
+2. Load the Kong Enterprise Trial Image:
+
+    ```
+    $ docker image load --input kong-enterprise-edition-ft-0.36.tar.gz 
+    ```
+
+3. Run `docker images` to find the ID for the Kong Enterprise
+image.
+
+4. Tag the image ID for easier use in the commands that follow:
+
+    ```
+    $ docker tag <IMAGE_ID> kong-ee
+    ```
+
+5. Create a Docker network (_optional_)
+
+    Containers require a network in order to discover and communicate with each
+    other. If you would like to use this functionality, you can create a network
+    using the following command, replaying `kong-ee-net` with whatever you would
+    like to name your network:
```

Comment:
Containers require a network in order to discover and communicate with each other. To use this functionality create a network using the following command, replacing kong-ee-net with the name of your network: